noir-projects/aztec-nr/aztec/src/context/interface.nr

```diff
@@ -31,8 +31,8 @@ trait PublicContextInterface {
     fn push_nullifier_non_existent_read_request(&mut self, nullifier: Field);
     fn message_portal(&mut self, recipient: EthAddress, content: Field);
     fn consume_l1_to_l2_message(&mut self, content: Field, secret: Field, sender: EthAddress);
-    fn accumulate_encrypted_logs<N>(&mut self, log: [Field; N]);
-    fn accumulate_unencrypted_logs<T>(&mut self, log: T);
+    fn emit_unencrypted_log<T>(&mut self, log: T);
```

Comment:
IIUC please add a comment with a todo+ticket that in the end only one of these should remain.